vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_wallet.h

```diff
@@ -24,23 +24,24 @@ class UpholdWallet {
   void Generate(ledger::ResultCallback callback);
 
  private:
-  void OnGenerate(
-      const type::Result result,
-      const User& user,
-      ledger::ResultCallback callback);
+  void OnGenerate(const type::Result result,
+                  const User& user,
+                  ledger::ResultCallback callback);
 
-  void OnCreateCard(
-      const type::Result result,
-      const std::string& address,
-      ledger::ResultCallback callback);
+  void OnCreateCard(const type::Result result,
+                    const std::string& address,
+                    ledger::ResultCallback callback);
 
-  type::WalletStatus GetNewStatus(
-      const type::WalletStatus old_status,
-      const User& user);
+  void OnTransferTokens(const type::Result result,
+                        std::string drain_id,
```

Comment:
Seems like `drain_id` should be `const std::string&` for consistency.

Reply:
Resolved in e7a597ba21dd2e98bb46cc6f8791c33c848e6525.